index.html

```diff
@@ -226,7 +226,47 @@ <h3 class="list_text">Skills</h3>
     </section>
     <!-- about section ends -->
 
+    <!-- Contact Form section starts here -->
+    <section class="form-container">
+        <div class="img_geometry">
+            <img src="img/image_geometry_1.png" alt="" class="shape1">
+        </div>
+        <header class="title_font">Get Started</header>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Exercitationem, odio? Lorem ipsum dolor sit amet.</p>
+        <form class="form" action="https://formspree.io/f/xqkjgdvv" method="POST">
+            <ul>
+                <li>
+                    <input type="text" name="name" placeholder="Your Name" maxlength="30" required>
+                </li>
+                <li>
+                    <input type="email" name="email" placeholder="Your Email" required>
+                </li>
+                <li>
+                    <textarea name="message" placeholder="Enter your message here..." cols="25" rows="5" maxlength="500"></textarea>
```

Comment:
**_as part of the requirements kindly fix these issues with the `textarea` :_** 

- [ ] kindly add the HTML validation `required` to your textarea input
- [ ] to match the Figma design kindly fix your text area padding, since its overlapping from the right , I suggest this solution : 

```CSS
.form textarea {
    width: 288px;
    height: 112px;
    margin-top: 8px;
    /* start solution*/
    /* remove padding-left: 16px; */
    /* remove padding-top: 15px; */
    /* add overall padding*/
    padding: 10px;
    /* end solution*/
    background: #fff;
    border: 1px solid #dfdfe0;
    border-radius: 4px;
    font-family: 'Poppins', sans-serif;
    font-style: normal;
    font-weight: 400;
    font-size: 14px;
    line-height: 21px;
    color: #7d7c84;
    transition: 0.5s;
}
```

Before           | After
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/199472563-91202cd8-9830-4ce8-9f15-57bf430aa29e.png) | ![image](https://user-images.githubusercontent.com/89282221/199473410-9148a3e3-60dd-4214-b92e-6565c28fd4bf.png)